sycl/test-e2e/Graph/Explicit/update_with_indices_accessor.cpp

```diff
@@ -0,0 +1,71 @@
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+// Extra run to check for leaks in Level Zero using UR_L0_LEAKS_DEBUG
+// RUN: %if level_zero %{env UR_L0_LEAKS_DEBUG=1 %{run} %t.out 2>&1 | FileCheck %s %}
+//
+// CHECK-NOT: LEAK
+// Unsupported: opencl, level_zero
+
+// Tests updating a graph node scalar argument using index-based explicit update
+
+#include "../graph_common.hpp"
+
+int main() {
+  queue Queue{{sycl::ext::intel::property::queue::no_immediate_command_list{}}};
+
+  if (!are_graphs_supported(Queue)) {
+    return 0;
+  }
+
+  const size_t N = 1024;
+
+  exp_ext::command_graph Graph{
+      Queue.get_context(),
+      Queue.get_device(),
+      {exp_ext::property::graph::assume_buffer_outlives_graph{}}};
+  std::vector<int> HostDataA(N, 0);
+  std::vector<int> HostDataB(N, 0);
+
+  buffer BufA{HostDataA};
+  buffer BufB{HostDataB};
+  BufA.set_final_data(nullptr);
+  BufB.set_final_data(nullptr);
+
+  exp_ext::dynamic_parameter InputParam(BufA.get_access(), Graph);
+
+  auto KernelNode = Graph.add([&](handler &cgh) {
+    // Register the input pointer, we should be using set_arg but can't
+    // currently test that with CUDA
+    // cgh.set_arg(0, ptrA)
```

Comment:
```suggestion
    // cgh.set_arg(0, BufA)
```
I don't think this comment is accurate.

Reply:
Good catch, though I think it should be `Acc` in the comment so I've updated it as such.